src/modules/order-detail/order-detail.controller.ts

```diff
@@ -51,4 +53,12 @@ export class OrderDetailController {
   remove(@Param('id') id: number) {
     return this._orderDetailService.remove(+id);
   }
+
+  @ApiOperation({ summary: 'Update order detail' })
+  @ApiOkResponse({ type: OrderDetailResponseDto })
+  @UseGuards(AuthGuard('jwt'))
+  @Put()
```

Comment:
Generally, the put method receive the ID to update in params

Ex
@Put(':id')
update(@Param('id') id: number)